```
 FAIL  src/Dog.test.tsx
  ● Test suite failed to run

    Cannot find module 'react-testing-library/cleanup-after-each' from 'Dog.test.tsx'

    > 1 | import 'react-testing-library/cleanup-after-each';
        | ^
      2 | import 'jest-dom/extend-expect';
      3 | 
      4 | import { Dog } from './Dog';

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:221:17)
      at Object.<anonymous> (src/Dog.test.tsx:1:1)
```
I guess it's missing react-testing-library as a dependency.  
(the CI/CD build succeeds because tests are not run, at the moment..)

--- a/package.json
+++ b/package.json
@@ -29,6 +29,8 @@
     "not op_mini all"
   ],
   "devDependencies": {
-    "@types/react-bootstrap": "^0.32.17"
+    "@types/react-bootstrap": "^0.32.17",
+    "jest-dom": "^3.1.3",
+    "react-testing-library": "^6.0.4"
   }
 }

